mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -173,6 +171,8 @@
     <!-- Preference for developers -->
     <string name="preferences_remote_debugging">Remote debugging via USB</string>
     <!-- Preference title for switch preference to show search suggestions -->
+    <string name="preferences_show_search_shortcuts">Show search shortcuts</string>
```

Comment:
Comment should be updated to "(...)show search shortcuts".

Reply:
Good catch, thanks!